packages/core/content-manager/admin/src/preview/components/PreviewHeader.tsx

```diff
@@ -177,32 +216,50 @@ const PreviewHeader = () => {
   };
 
   return (
-    <Flex
-      justifyContent="space-between"
+    <Grid.Root
+      gap={3}
+      gridCols={3}
+      paddingLeft={2}
+      paddingRight={2}
       background="neutral0"
-      padding={2}
       borderColor="neutral150"
       tag="header"
     >
-      <Flex gap={3}>
-        <ClosePreviewButton />
-        <Typography tag="h1" fontWeight={600} fontSize={2}>
-          {title}
-        </Typography>
-        <DocumentStatus />
-      </Flex>
-      <IconButton
-        type="button"
-        label={formatMessage({
-          id: 'preview.copy.label',
-          defaultMessage: 'Copy preview link',
-        })}
-        onClick={handleCopyLink}
-      >
-        <LinkIcon />
-      </IconButton>
-    </Flex>
+      {/* Title and status */}
+      <Grid.Item xs={1}>
+        <Flex gap={3}>
+          <ClosePreviewButton />
+          <Typography tag="h1" fontWeight={600} fontSize={2}>
+            {title}
+          </Typography>
+          <DocumentStatus />
+        </Flex>
+      </Grid.Item>
+
+      {/* Tabs */}
+      <Grid.Item xs={1}>
+        <PreviewTabs />
+      </Grid.Item>
+
+      {/* Copy link */}
+      <Grid.Item xs={1} justifyContent={'end'}>
+        <IconButton
+          type="button"
+          label={formatMessage({
+            id: 'preview.copy.label',
+            defaultMessage: 'Copy preview link',
+          })}
+          onClick={handleCopyLink}
+        >
+          <LinkIcon />
+        </IconButton>
+      </Grid.Item>
+    </Grid.Root>
   );
 };
 
+const StatusTab = styled(Tabs.Trigger)`
+  text-transform: uppercase;
+`;
```

Comment:
This isn't necessary with the right translation keys

Reply:
At some point this compo needs to  uppercase  it, as it compares the document status with the Status tab value. Do you think it would be preferable to upercase it [here](https://github.com/strapi/strapi/pull/22025/files#diff-4483fd0983d09b3ed6d90b4f62c82777caad79bc82ade8b242fbb3abe652f1d0R168)

Reply:
I believe it's only the `value` props (+ `onValueChange`) that need to match, what you render inside the trigger should be purely visual and disconnected from the logic

Reply:
true! alright :) last questoin then, do I create a new key for this?  as it looks like content-manager.containers.List.(published|draft) are not upper cased

Reply:
The value seems to be uppercased, for english at least: https://github.com/strapi/strapi/blob/5c463a4f3349c33b8034dcda66a04758d0efc542/packages/core/content-manager/admin/src/translations/en.json#L104-L106

Reply:
Oooh sorry I thought your CSS was only capitalizing the first letter! (because that's done in a few places in the CM and I don't like it 😁)

But yes for this your code is perfect 💯